dist/images/ovnkube.sh

```diff
@@ -803,8 +803,12 @@ function memory_trim_on_compaction_supported {
 function get_node_zone() {
   zone=$(kubectl --subresource=status --server=${K8S_APISERVER} --token=${k8s_token} --certificate-authority=${K8S_CACERT} \
      get node ${K8S_NODE} -o=jsonpath={'.metadata.labels.k8s\.ovn\.org/zone-name'})
-  if [ "$zone" == "" ]; then
-    zone="global"
+  if [ -z "$zone" ]; then
+    if [[ ${ovn_enable_interconnect} == "true" ]]; then
```

Comment:
I know this may be controversial because it would disrupt the upgrade detection, right? One solution for that would be to also consider the number of nodes per zone -- if such info was made available to `ovnkube.sh`. That way we could safely determine that the label is not present and we still want to go ahead with ovn-ic.